org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/extensions/contentmodel/XMLSchemaDiagnosticsTest.java

```diff
@@ -27,6 +27,18 @@
  */
 public class XMLSchemaDiagnosticsTest {
 

```

Comment:
so right now PrematureEOF doesn't generate an error for empty docs, which is debatable, but let's say we keep that behavior for now. At least the test methods should be changed to *NoError, since no diagnostic is expected